docs/nuget-org/package-readme-on-nuget-org.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Package Readme on NuGet.org
+description: Detailed explanation of how Readme files on NuGet.org are rendered and what to do when you run into issues.
+author: chgill-MSFT
+ms.author: chgill
+ms.date: 02/23/2021
+ms.topic: conceptual
+ms.reviewer: anangaur
+---
+
+# Package Readme on NuGet.org
+
+[Include a Readme file in your NuGet package](https://docs.microsoft.com/nuget/reference/msbuild-targets#packagereadmefile) to make your package details richer and more informative for your users!
+
+This is likely one of the first elements users will see when they view your package details page on NuGet.org and is essential to making a good impression!
+
+## What should my Readme include?
+
+Consider including the following items in your readmes:
+* An introduction to what your package is and does - what problems does it solve?
+* How to get started with your package - are there any specific requirements?
+* Links to more comprehensive documentation if not included in the readme itself.
+* At least a few code snippets/samples or example images.
+* Where and how to leave feedback such as link to the project issues, Twitter, bug tracker, or other platform.
+* How to contribute, if applicable.
+
+Keep in mind, high quality readmes can come in a wide variety of formats, shapes, and sizes! If you already have a package available on NuGet.org, chances are that you already have a `readme.md` file at in your repository that would be a great addition to your NuGet.org details page.
```

Comment:
Hmmm do we most GitHub/repo READMEs should just be copy-pasted to the package readme? Or is the content/intent a bit different? I feel that a package readme should be more tailored toward consumption and usage via NuGet whereas a repo README might have more about contribution or GitHub operations. But maybe I am overthinking it.

Reply:
I changed it to:

> Keep in mind, high quality readmes can come in a wide variety of formats, shapes, and sizes! If you already have a package available on NuGet.org, chances are that you already have a `readme.md` or other documentation file in your repository that would be a great addition to your NuGet.org details page.

When I look at readmes for popular packages, I see a mixed bag of whether or not the default repo readme is appropriate for NuGet.org. I added "or other documentation file" to be more generalized 😊

Reply:
`README.md` 😛